### DISCUSSION TITLE
[KtonDAO]New staking model for KTON DAO

### DISCUSSION THREAD

**Discussion by SasoLithops** [ktondao/KtonDAO#30] [Q&A]
Hi KTON hodlers!

IMPORTANT: I GOT SOME FEEDBACK MY PROPOSAL IS HARD TO UNDERSTAND IF YOU DON'T LIKE MATH AND I HAVE LEARNED MY EDITING SKILLS ARE REALLY BAD. OUR AMAZING ADVOCATE LEPHOFORA MADE IT SO MUCH BETTER, SO PLEASE SCROLL A BIT LOWER TO SEE HIS VERSION IF MINE IS TO HARD TO READ.

I have been thinking on this idea for a while. I want your opinion on this and what you think should be done better and if you find it sustainable and good for us holders/stakers/LP providers.

**_CURRENT STAKING MODEL:_**

Until 25th or 26th September 2024 there is/will be 152 702 246 Ring minted.
That is 418 362 Ring per day.
Out of that:

- Treasury gets 251 017 Ring per day
- **KTON stakers get 83 672 Ring per day**
- RING stakers get 83 672 Ring per day

After 26th or 27th September there will be 167 131 170 RING inflation for 2024/2025 so rewards will soon go slightly up.

Huge thank you to Laki for providing me with above numbers:

So important for us is to know now KTON DAO gets 83 672 Ring per day which are distributed to KTON stakers. So at this time KTON stakers get all the rewards from KTON DAO.

**_IDEA FOR NEW STAKING MODEL_**

Bear with me please till the end of idea I know some stakers will probably not like it, but we must take few steps back to continuously run forward.

So let us focus on what we mentioned above. KTON DAO gets 83 672 Ring per day. That is 100% funds from KTON DAO are given to KTON stakers, That would mean that KTON DAO can not fund itself and is always dependant of other ways of funding (Ring treasury).

**What if:**

First we ask for 10% from what Ring treasury gets to go to KTON DAO. That would mean:

- Treasury gets 225 915 Ring per day
- **KTON stakers get 108 773  Ring per day**
- RING stakers get 83 672 Ring per day

Of course for asking for this 10% we will in the end have to give something back to Treasury.

Now remember when I sad we must take few steps back.

Let us asume that proposal gets passed and KTON DAO gets 108 773 RING per day. 
That is new 100 %.

What if 30% of that ammount now goes to KTON DAO treasury. 
That is 70% which comes to 76 141 RING goes to stakers and 30% which is 32 632 RING goes to KTON DAO treasury. Let me explain further.

- _We could DCA into KTON once a week/month with 20% of KTON DAO treasury ammount. That should get us quite a lot of KTON(cca_ 150 first month) in KTON DAO treasury and probably push price up. But why would it go up. Now here comes the best part. Where do stakers get BTC for staking native token. In KTON staking. 
- _40% of KTON DAO treasury ammount could go for DCA into BTC._ I don't want to go into even deeper math(unless you want me too please post in comments) 25% of that is given back to stakers(I have checked WBTC(wormhole Moonbeam) and Interlay iBTC(my favourite) are both on Hydration where we have good Ring liquidity). 25% would be given to KTON/Ring LP farmers 25% are given to Ring treasury for giving up on 5% of their current supply and 25% remain in KTON DAO treasury. 
- _40% that remain I would leave as Ring_. 50% of that would go to KTON/Ring farmers rest would be left for KTON DAO treasury.

So the new KTON DAO staking daily model would be:

**KTON STAKERS:** 76 141 RING + BTC worth of 3 263 RING = 79 404 RING (was 83 672 Ring)
**KTON/Ring LP Farmers:** BTC worth of 3 263 RING + 6 526 RING = 9 789 (was 0 Ring) 
   We needed to ask from Ring Treasury now this will bring 1 762 133 RING per half a year and now we got 1 400 000 RING.
**KTON DAO:** KTON buyback worth of 6 526 RING + BTC worth of 3 263 RING + 6 526 RING = 16 316 RING (was 0 Ring)
Return to RING treasury: BTC worth of 3 263 RING (they let go of 25 101 RING) 

Thank you for going through this. Hope it makes sense.

It would lower staking rewards by 5.1% but next to RING you now earn BTC.
KTON DAO treasury is self funded and it has RING, KTON and BTC.
LP farmers get great rewards.
We give BTC back to Ring treasury.

This should definitely boost KTON imho. But good usecases should be provided for RING too to make all this sustainable. I think since RING is the gas and everything token of Darwinia that should be easier. Let me know what you all think.

########################################################################################################
LEPHOFORA:

I really love this proposal, and it looks perfect to me.

Although I think it may be a little bit difficult for some to understand, and to help understand this proposal, I'll break it down into simpler terms:

This breakdown is subject to corrections if any.

Current Staking Model:

- Right now, Darwinia mints 418,362 RING tokens every day.
- This daily minting is split into three parts:

  - 251,017 RING goes to the Treasury.

  - 83,672 RING goes to KTON stakers (people who stake KTON tokens).

  - 83,672 RING goes to RING stakers (people who stake RING tokens).

Proposed New Staking Model:

The proposal suggests making some changes to how these rewards are distributed to create a more sustainable system.

1. Take 10% from the RING Treasury: 

   - The Treasury currently gets 251,017 RING per day. The proposal suggests that 10% of this (25,101 RING) should go to the KTON DAO (the community managing KTON).

2. Redistribute Funds:

   - With this new model, the funds will be split differently:
     - 225,915 RING goes to the Treasury (down from 251,017).
     - 108,773 RING goes to KTON stakers (up from 83,672).
     - 83,672 RING goes to RING stakers (remains the same).

3. KTON DAO's New Distribution:

   - Once KTON DAO gets its new share (108,773 RING per day), it proposes to:
     - Keep 30% (32,632 RING) in the KTON DAO Treasury to fund itself.
     - Give 70% (76,141 RING) to KTON stakers.

4. Diversify Investments:

   - The KTON DAO would invest in KTON and BTC (Bitcoin) to strengthen its Treasury:
     - 20% of the funds would be used to buy KTON.
     - 40% would be used to buy BTC.
     - 40% would be kept as RING.
    
5. New Rewards Structure:

   - With this new model:
     - KTON Stakers would earn slightly less RING (76,141 RING), but they would also earn BTC worth 3,263 RING. So in total, they'd get rewards worth 79,404 RING.

     - KTON/RING LP Farmers (those providing liquidity by pairing KTON and RING) would start earning rewards, including BTC and RING, totaling 9,789 RING.

     - KTON DAO Treasury would receive funds from KTON buybacks, BTC, and RING, totaling 16,316 RING.
     - RING Treasury would also receive some BTC back as a return.

Why This Matters:

- KTON Stakers: You'll still get most of your RING rewards, but now you'll also earn some BTC, which could be valuable.

- KTON DAO: The DAO will have a steady income, making it more sustainable and less dependent on outside funding.

- LP Farmers: This proposal introduces new rewards for those providing liquidity, which might encourage more people to do so.

- RING Treasury: Though it gives up some RING, it will receive BTC in return, diversifying its assets.

The Trade-Off:

- KTON stakers would earn about 5.1% less in RING, but this would be offset by earning BTC.

- The proposal aims to boost the value of KTON by buying it regularly and also creating a stronger, self-funded DAO.

Key Considerations:

- This new model reduces the immediate RING rewards slightly but adds BTC to the mix, potentially increasing the overall value of staking.

- It creates a more balanced system where KTON DAO can sustain itself and LP farmers are incentivized with new rewards.

Overall, this proposal aims to make the KTON ecosystem more sustainable, diversified, and potentially more profitable in the long run.

**Comment by nblogist** [comment]
> So let us focus on what we mentioned above. KTON DAO gets 83 672 Ring per day. That is 100% funds from KTON DAO are given to KTON stakers, That would mean that KTON DAO can not fund itself and is always dependant of other ways of funding (Ring treasury).

I like the cause for this proposal, the numbers look okay to me, but maybe we need to discuss more before we finalize them- but in general: KTON DAO getting funded in similar method is a good approach.

**Reply by SasoLithops** [reply]
Absolutely discuss more. This is just a spark 😉

**Comment by hackfisher** [comment]
Firstly, governance proposals within KtonDAO can only determine the allocation of the KtonDAO Treasury. If there is a need to propose a preconditioned proposal to RingDAO or the Darwinia Treasury, it is recommended to separate them since they are voted on by different groups.

Overall, I support diversifying the assets and usage held by KtonDAO, but from the perspective of a RING holder, I oppose the Darwinia Treasury allocating more RING to KtonDAO, as it doesn’t benefit the RING token.

It would be helpful if the proposal could be described more clearly, including specifying to whom the proposal is being addressed.

**Reply by SasoLithops** [reply]
Thank you for your opinion. Much appreciated. This was just initial idea so I see how KTON holders feel about this idea and how we could move this forward. 

Any constructive critical opinion is more than welcome as this only makes it better. 

Main point of this proposal was to make KTON staking different than any other. With KTON buybacks(also potential burning if community feels that way, I don't) and BTC rewards I feel we could achive that.
 I will get some more data that can be pulled from chain with help of @AurevoirXavier to be more specific next time and will definitely take into account to separate two proposals one for RING DAO and one for KTON DAO. 

Sadly at the moment it feels like no one cares much but I plan to push this forward as see benefits to us KTON holders and as I feel it will benefit RING too, but need more data and numbers to prove it.

**Reply by sonoferin2000** [reply]
I disagree with the idea that this wouldn’t benefit RING stakers, particularly since many of them are also KTON stakers. This proposal could encourage more RING to be locked up in order to generate additional KTON rewards, which could then be staked rather than sold off, helping to maintain the price. It also benefits liquidity providers, making it a strong proposal in my opinion. This creates a closed-loop system that benefits everyone.

I’m also open to the DAO holding the entire additional amount to fund future initiatives. This could establish a continuous stream of funding, such as for developing dApps that utilize both RING and KTON. Even if it’s something as simple as a gambling app, we need more options for using our RING and KTON beyond just staking. We can’t rely on the project to provide endless handouts, but I believe that a one-time token allocation or a small, steady flow of tokens could help us launch new ventures.

**Reply by hackfisher** [reply]
> This proposal could encourage more RING to be locked up in order to generate additional KTON rewards

RingDAO/Treasury already give 20% the new issuance to KtonDAO to incenvtive th lock by Kton Staking, the marginal effect is low, more RING spending on this will cause more deficit which is not good for RING holders, the amount is small, but I can not agree the logic and philosophic under hood.

As a KTON holder too, I encourage KtonDAO to be more independent.

Disclaimer: My opinions are my own and do not represent Itering. I will only use my personal votes if the proposal is proposed.

**Reply by sonoferin2000** [reply]
I would also like to see KtonDAO be more independent, and hope that as a community,we can put our heads together and brainstorm some actionable ways in which we can reach that goal.

**Reply by SasoLithops** [reply]
Awesome. @sonoferin2000 check this: https://github.com/orgs/darwinia-network/discussions/1573

Yours and everyone reading this opinion much appreciated.

**Comment by SasoLithops** [comment]
I did some more research. Just writing down what will be added in final proposal. For buying BTC we can use Hydration DCA. That is what Polkadot does with buying USDC and USDT for treasury. Will explain how it works in next proposal.

Maybe Snowswap can do the same for buying back KTON. Partnership Dude can you reach out to them and ask if they familiar with Hydration DCA and if they can replicate it?

**Comment by hackfisher** [comment]
Contract code for your reference:

https://github.com/darwinia-network/KtonDAO/blob/d05e05d56682651110a40384fa6d8fadc2516305/src/staking/KtonDAOVault.sol#L44-L75

**Comment by sonoferin2000** [comment]
I really like your idea on this in general,the regular buying of kton by the dao would help stabalise the price of kton. and the addition of BTC into the mix would be a welcome addition. hopefully it will remain within the darwinia egosystem and not be syphoned off to another chain, we need to retain value here. i am supportive of this idea.

**Reply by SasoLithops** [reply]
Thank you for your opinion. You are OG of KTON DAO. If it wasn't for you and our disscussions in KTON DAO Github thread I would probably give up on the way. 

I think we need to change something and make it more attractive than it is now. 
This would imho benefit both Ring and KTON holders too like you say, as KTON would be needed to get all the staking rewards and like you said only way to get it is lock more Ring(beneficial for RING holders as RING gets taken from the market for a period of time) or buy it on the market(good for KTON holders as less KTON on market)
It would lower staking rewards when more join....but I would much rather have a bit lower staking rewards and KTON at higher price....as once it starts pumping in price, people see it in coingecko and start asking about it(best free marketing).
